### PR TITLE
Improve Typography System

### PR DESCRIPTION
**Pull request by endigo9740**
Addresses #1189

As well as:

Addresses #1376
Addresses #1356

### CODE REVIEW

File: src/routes/(inner)/components/autocomplete/+page.svelte [hunk 1]
```diff
@@ -222,9 +222,9 @@ const flavorOptions: AutocompleteOption[] = [
 		<section class="space-y-4">
 			<h2>Input Chip</h2>
 			<p>We've provided a demo of using Autocomplete alongside a Skeleton Input Chip component below.</p>
-			<pre>inputChip: {JSON.stringify(inputChip, null, 2)}</pre>
-			<pre>flavorOptions: {JSON.stringify(flavorOptions, null, 2)}</pre>
-			<pre>inputChipList: {JSON.stringify(inputChipList, null, 2)}</pre>
+			<!-- <pre>inputChip: {JSON.stringify(inputChip, null, 2)}</pre> -->
```
Comment (bartduisters) [inline src/routes/(inner)/components/autocomplete/+page.svelte:225]: Is this on accident?
If they are no longer necessary, could be removed

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <script lang="ts">
 	// Dependency: Highlight JS
 	import hljs from 'highlight.js';
-	import '$lib/styles/highlight-js.css'; // was: 'highlight.js/styles/github-dark.css';
+	import '$docs/styles/highlight-js.css'; // was: 'highlight.js/styles/github-dark.css';
```
Comment (bartduisters) [inline src/routes/+layout.svelte:5]: The comment should now be `// was: '$lib/styles/highlight-js.css'`, or be removed - is it important what it used to be?

File: src/routes/(inner)/components/autocomplete/+page.svelte (hunk 1 above)
Comment (endigo9740) [reply src/routes/(inner)/components/autocomplete/+page.svelte:225]: I need to check the git blame on this, but someone left these in and it made it all the way to production darn it. You're right though, should just be axed. Handling that now.

File: src/routes/+layout.svelte (hunk 2 above)
Comment (endigo9740) [reply src/routes/+layout.svelte:5]: The comment isn't really useful any more so I'm dropping it completely.

**Comment by endigo9740** [comment]
@bartduisters thanks, I've implemented changes based on your suggestions above.

**Pull request merged**